PrestaShop/PrestaShop
# Fix email theme permission

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Add missing AdminMailTheme parent tab rights
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | yes
| Deprecations? | no
| Fixed ticket? | Fixes #16337
| How to test?  | See below

## How to test?
There is two ways to test that PR.

1. After a fresh install on branch 1.7.6.x
2. After an upgrade from 1.7.6.2 to 1.7.6.x

Either case, you should be able to change _Email Themes_ permissions in _configure > Advanced parameters > Team > Permissions_




---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/16696)
